corehq/apps/integration/kyc/models.py

```diff
@@ -202,14 +206,15 @@ def kyc_is_verified(self):
     def kyc_provider(self):
         return self.user_data.get('kyc_provider')
 
-    def update_verification_status(self, is_verified, device_id=None):
+    def update_verification_status(self, is_verified, device_id=None, error_message=None):
         from corehq.apps.hqcase.utils import update_case
 
         assert is_verified in [True, False]
         update = {
             'kyc_provider': self.kyc_config.provider,
             'kyc_last_verified_at': datetime.utcnow().isoformat(),  # TODO: UTC or project timezone?
             'kyc_is_verified': str(is_verified),
+            'kyc_verification_error': error_message if error_message else '',
```

Comment:
Should we stay consistent with using None or empty string when not present?

Reply:
@mkangia What do you mean? `kyc_verification_error` will be `''` when `error_message` is `None`.

Reply:
Ah, got it. Didn't realize this function itself was being called from the other place.

Reply:
Actually, is `None` not okay to be saved for this? Just wondering the need for this if statement here.